lib/proxy-agent.js

```diff
@@ -136,14 +134,12 @@ class ProxyAgent extends DispatcherBase {
     )
   }
 
-  async [kClose] () {
-    await this[kAgent].close()
-    await this[kClient].close()
+  [kClose] () {
+    return Promise.all([this[kAgent].close, this[kClient].close])
```

Comment:
```suggestion
    return Promise.all([this[kAgent].close(), this[kClient].close()])
```
are they functions, they aren't?

Reply:
Uff right.